memory/migrate_pages.py

```diff
@@ -86,9 +93,14 @@ def test(self):
         cmd = './node_move_pages -n %s' % self.nr_chunks
 
         if self.hpage:
-            cmd += ' -h'
-            if self.hpage_commit:
-                cmd += ' -o'
+           if (self.dist.name == 'rhel' and self.dist.version == '9'):
+               if self.hpage_commit:
+                  self.cancel("Hugepage(over commit) tests are cancelled on RHEL-9")
```

Comment:
Introduces a code style warning for >80 characters. The if hpage_commit block isn't necessarily required. Cancel the test if hpage is set scenario will also cover hpage_commit case as well.